website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4578,6 +4579,7 @@ firings:
 {{< code_sample "sdks/python/apache_beam/examples/snippets/snippets_test.py" model_early_late_triggers >}}
 {{< /highlight >}}
 
```

Comment:
Since you've added snippets, I assume the intent is to include the appropriate block so they get displayed: 

```
{{< highlight go >}}
{{< code_sample "sdks/go/examples/snippets/09triggers.go" model_early_late_triggers >}}
{{< /highlight >}}
```
And similar throughout.

You can see how things render using `./gradlew :website:serveWebsite ` to stand up  a localhost version of the website. Note, for new snippet files you need to add at least one code_sample block for them to be known. I end up iterating between using the gradle command above, stopping that and then clearing the running docker containers `docker stop $(docker ps -q -f ancestor=beam-website)` to see fresh changes on the next run of the gradle command.